lectures/lp_intro.md

```diff
@@ -40,6 +40,11 @@ We provide a standard form of a linear program and methods to transform other fo
 
 We tell how to solve a linear programming problem using [SciPy](https://scipy.org/) and [Google OR-Tools](https://developers.google.com/optimization).
 
+```{seealso}
+In another lecture, we will employ the linear programming method to solve the 
+{doc}`optimal transport problem <intro:opt_transport>`.
```

Comment:
```suggestion
{doc}`optimal transport problem <intermediate:opt_transport>`.
```

Reply:
@shlff are you trying to refer to the lecture in the current intermediate lecture series?

Reply:
Yep thanks @mmcky . 

If I want to do cross-series reference, I wonder where I can find the name for the different lecture series, 

e.g., the name `intermediate` for the intermediate lecture series, is it contained in the `.yml` file?